app/views/notifications/_lending_accepted_notification.html.erb

```diff
@@ -0,0 +1,39 @@
+<!-- Button trigger modal -->
+<button type="button" class="btn btn-notification" style= "background-color: <%= notification.unread == true ? "#cce4ec" : "#dddfe0c" %> !important" data-bs-toggle="modal" data-bs-target="#exampleModal">
```

Comment:
The data-bs-target has to be the same as for the other notification, it has to be custom for every notification, otherwise there will be a bug where each button shows the same modal, because they target the first modal with this id. The id also hase to be changed away from "exampleModal" to id="<%= "notificationModal#{notification.id}" %>"

Reply:
Did the change after talking to jacob